sdk/eventhub/event-hubs/samples/v5/browser/package.json

```diff
@@ -29,12 +31,16 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/event-hubs": "^5.0.0",
-    "@azure/identity": "^2.0.1"
+    "@azure/identity": "^2.0.1",
+    "buffer": "^6.0.3",
```

Comment:
Do we need to list buffer and process as direct dependencies even though they're transitive dependencies of event hubs?

Reply:
You are right. they are not needed.